Makefile

```diff
@@ -31,7 +31,8 @@ implementations := \
 	rotational-cipher \
 	perfect-numbers \
 	change \
-	sieve
+	sieve \
+	affine-cipher
```

Comment:
In developing binary-search, I was super annoyed when tests would fail at the end of the build. From that experience I'd recommend instead putting new problems at the beginning of the list to fail early!

Reply:
Is that something to keep in mind for the future, or should I do that in this PR too.

Reply:
Yeah for sure